pkg/controller/workloadspread/workloadspread_controller_test.go

```diff
@@ -1620,6 +1620,201 @@ func TestDelayReconcile(t *testing.T) {
 	}
 }
 
+func TestManagerExistingPods(t *testing.T) {
+	cases := []struct {
+		name                 string
+		getPods              func() []*corev1.Pod
+		getNodes             func() []*corev1.Node
+		getCloneSet          func() *appsv1alpha1.CloneSet
+		getWorkloadSpread    func() *appsv1alpha1.WorkloadSpread
+		expectWorkloadSpread func() *appsv1alpha1.WorkloadSpread
+	}{
+		{
+			name: "manager the pods that were created before workloadSpread",
+			getPods: func() []*corev1.Pod {
```

Comment:
add a case for existing pods with injected workload spread annotation, but the spread in annotation cannot match the pod again

Reply:
done

Reply:
done